ui/src/dashboards/components/CellEditorOverlay.js

```diff
@@ -179,6 +183,7 @@ class CellEditorOverlay extends Component {
               onSelectGraphType={this.handleSelectGraphType}
               onCancel={onCancel}
               onSave={this.handleSaveCell}
+              isSavable={queriesWorkingDraft.every(isQuerySavable)}
```

Comment:
`OverlayControls` isn't used anywhere else and currently isn't slated to be used anywhere else.  Nor is it particularly reusable absent the `isSavable` prop.

In my experience, naming a boolean variable `isNotSomething` can be confusing.  I try to stear clear of double negatives.  'Is not something false' just doesn't sit well in my brain.  

The function is not checking exclusively if something is fully qualified.  Hence the name not being such.